cmd/nerdctl/ps.go

```diff
@@ -125,26 +130,41 @@ func printContainers(ctx context.Context, cmd *cobra.Command, containers []conta
 	if err != nil {
 		return err
 	}
+
 	var w io.Writer
 	w = os.Stdout
 	var tmpl *template.Template
 	format, err := cmd.Flags().GetString("format")
 	if err != nil {
 		return err
 	}
+
+	size := false
+	if !quiet {
+		size, err = cmd.Flags().GetBool("size")
+		if err != nil {
+			return err
+		}
+	}
+
 	switch format {
 	case "", "table":
 		w = tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
 		if !quiet {
-			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES")
+			if size {
+				fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES\tSIZE")
+			} else {
+				fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES")
+			}
 		}
 	case "raw":
 		return errors.New("unsupported format: \"raw\"")
 	case "wide":
 		w = tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
 		if !quiet {
-			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tPLATFORM\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES\tRUNTIME")
+			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tPLATFORM\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES\tRUNTIME\tSIZE")
 			wide = true
+			size = true
```

Comment:
useless based on this  https://github.com/containerd/nerdctl/pull/1177/files#diff-2ffc2cdeb00bd7b916b01ebdf34c9083512cf0057fdd7d886b794e4d5e510eccR144